harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -291,6 +298,22 @@ def cleanup_iterator() -> None:
                 self._run()
 
     def _run(self) -> None:
+        # Special code path only used for DeepSpeed Autotuning.
+        if self._dsat_mode:
+            ops = self.context._core.searcher.operations()
+            op = next(ops)
+            # TODO: remove `while True` in favor of exact number of steps which should run.
+            # TODO: also assuming here that op.length is always the number of *steps* to train for
+            # and never epochs or other units. Double check.
+            while True:
+                with dsat.dsat_reporting_context(core_context=self.context._core, op=op):
+                    # TODO: verify we're using `step_id` as intended.
```

Comment:
Can you delete this comment, and `step_id`?  It looks like it's no longer used for anything at all.

Reply:
Yup, I can delete the comment. What do you mean by also deleting `step_id`? You don't mean the parameter right?

Reply:
Removed the comment in #6988, but also didn't understand about removing `step_id` itself.

Reply:
when I look at the code, it looks like `step_id` is just not used.  The function accepts it and verifies it is greater than zero, then ignores it.  So I mean finding all instances of `step_id`, including the function parameter, and deleting them.

(Also verifying that it is actually useless of course, I checked pretty quickly)